src/services/appointmentData.service.js

```diff
@@ -54,28 +66,71 @@ export function sortData(data, sortKey) {
             return first - second;
         }
     });
-    return newData;
 }
 
 export function filterData(data, { filterByAvailable, filterByZipCode }) {
     return data.filter((d) => {
         if (filterByAvailable && !isAvailable(d)) {
             return false;
         }
-        if (
+        return !(
             filterByZipCode.zipCode &&
             !isWithinRadius(d, filterByZipCode.zipCode, filterByZipCode.miles)
-        ) {
-            return false;
-        }
-        return true;
+        );
     });
 }
 
 export function getAppointmentData() {
+    let testDataTransformed = null;
+
+    if (process.env.NODE_ENV === "development") {
```

Comment:
We should change this to != production

In reality it's only ever set when we're in prod, as opposed to always set to dev otherwise

Reply:
The tests were failing.  Now I think this happens because I changed "now" to be the timestamp on any test data file that might be in the way.

Reply:
Finally settled on adding an environment variable to `.env.test`